### ISSUE TITLE
[format-js] ES2015 Module support

### ISSUE THREAD

**Issue by DaleJefferson** [facebook/nuclide#329]
It seems all of the code mods work with require statements and not import statements, by default (with "add missing Requires") it adds duplicate requires for all of my import statements.

```
const React = require('React');

import React from 'react';
```

**Comment by johnislarry** [comment]
cc @kyldvs who maintains this

**Comment by kyldvs** [comment]
Yeah this would be a good feature to add behind an option. I maintain this plugin as a side-project, so PRs welcome :)

**Comment by zertosh** [comment]
And I'd be more than happy to approve it!
